snuba/web/views.py

```diff
@@ -310,28 +310,12 @@ def storage_delete(
         check_shutdown({"storage": storage.get_storage_key()})
         body = parse_request_body(http_request)
         try:
-            """
```

Comment:
putting the schema validation here would also make it more clear what the HTTP interface looks like.

AS-IS you would have to dig into the `delete_from_storage` to figure out the interface.